Modules/Loadable/Markups/Testing/Cxx/vtkMarkupsAnnotationSceneTest.cxx

```diff
@@ -37,22 +38,36 @@ int vtkMarkupsAnnotationSceneTest(int argc, char* argv[])
 {
   // Test reading in a Slicer4 MRML scene with legacy annotation nodes (created using Slicer-4.1.1):
   //
-  // --- All Annotations
-  //  |--- Fiducials List
-  //  | |--- F
-  //  | |--- F_1
-  //  | |--- F_2
-  //  |--- Ruler list
-  //  | |--- M
-  //  | |--- M_1
-  //  |--- ROI list
-  //  | |--- R
-  //  | |--- R_1
-  //  | |--- R_2
-  //  |--- List
-  //    |--- F_4
-  //    |--- F_3
   //
+  // All Annotations (vtkMRMLAnnotationHierarchyNode1) -> vtkMRMLAnnotationDisplayNode1
```

Comment:
Nitpick: The docstring seems a bit more challenging to visually parse, I am wondering if placeholder could be user (e.g `vtkMRMLAnnotationTextDisplayNode1` -> `A_TextDisplay_1`) ?